apis/python/src/tiledb/vector_search/ingestion.py

```diff
@@ -2661,8 +2693,9 @@ def consolidate_and_vacuum(
         logger.debug("Number of workers %d", workers)
 
         # Compute task parameters for main ingestion.
+        SCALED_VECTORS_PER_WORK_ITEM = int((VECTORS_PER_WORK_ITEM * 128) / dimensions / np.dtype(vector_type).itemsize)
```

Comment:
This is a bit confusing, maybe we can set the default partition size in bytes instead.

Reply:
Refactored